metrics/web/docs/features/public_dashboard/diagrams/public_dashboard_sequence_diagram.puml

```diff
@@ -0,0 +1,60 @@
+@startuml public_dashboard_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant Browser
+participant LoadingPage
+participant AuthNotifier
+participant FeatureConfigNotifier
+participant NavigationNotifier
+participant CoolPage
+participant LoginPage
+
+Bob -> Browser ++:  opens a "Cool Page" URL
+Browser -> LoadingPage ++: navigates to initialize all components
+LoadingPage --> Browser: returns the LoadingPage UI
+Browser --> Bob: sees the LoadingPage UI
+
+LoadingPage -> AuthNotifier ++: addListener(_authNotifierListener)
+AuthNotifier --> LoadingPage --: the listener is added
+LoadingPage -> AuthNotifier ++: subscribeToAuthenticationUpdates()
+AuthNotifier --> LoadingPage --: subscribed to auth updates
+
+LoadingPage -> FeatureConfigNotifier ++: addListener(_featureConfigNotifierListener)
+FeatureConfigNotifier --> LoadingPage --: the listener is added
+LoadingPage -> FeatureConfigNotifier ++: initializeConfig()
+FeatureConfigNotifier --> LoadingPage --: the config initialization is started
+
+AuthNotifier --> AuthNotifier ++: receives the user state
+AuthNotifier -> NavigationNotifier ++: handleAuthUpdates(authState)
+NavigationNotifier --> AuthNotifier --: the auth updates are handled
+AuthNotifier --
+
+FeatureConfigNotifier --> FeatureConfigNotifier ++: receives the feature config
+FeatureConfigNotifier --> LoadingPage: the feature config is initialized
+FeatureConfigNotifier --> AuthNotifier --: handlePublicDashboardFeatureConfigUpdates(model)
+AuthNotifier++
+
+alt publicDashboardFeature == true && authState == AuthState.logout
+AuthNotifier --> AuthNotifier: signInAnonymously()
+end
+
+AuthNotifier --> LoadingPage --: the auth is initialized
+LoadingPage --> NavigationNotifier ++: handleAppInitialized(true)
+
+alt page is available for the anonymous user
+NavigationNotifier --> CoolPage ++: push to the "Cool Page"
+CoolPage --> Browser --: returns the CoolPage UI
+Browser --> Bob: sees the CoolPage UI
+else page is not available for the anonymous user
+NavigationNotifier --> LoginPage ++: push to the "Login Page"
```

Comment:
🔨 
I believe it would be better to push to the dashboard page in this case. What do you think?